spark-screenshot-testing/src/test/kotlin/com/adevinta/spark/PaparazziUtils.kt

```diff
@@ -40,7 +40,7 @@ internal fun Paparazzi.sparkSnapshot(
 ): Unit = snapshot(name) {
     // Behave like in Android Studio Preview renderer
     CompositionLocalProvider(LocalInspectionMode provides true) {
-        SparkTheme {
+        SparkTheme(useLegacyStyle = false) {
```

Comment:
This means we are actually producing golden images with non-official specs?

Reply:
No, since it's the contrary, we produce golden image with the official specs and I plan to make it possible on the demo app to view the legacy style